app/views/submissions/_react_file_manager_boot_table.js.jsx.erb

```diff
@@ -59,7 +59,8 @@
       this.setState({ selected_files: [] });
 
       jQuery.ajax({
-        url: '<%= populate_file_manager_react_assignment_submissions_path %>',
+        url: '<%= populate_file_manager_react_assignment_submissions_path(
+          path: @path) %>',
```

Comment:
Okay.  I'm happy to change it.  Here is what I was using as a model from submissions/_repo_files_table.js.jsx.erb.  (Oddly enough, the form you recommend isn't used as much
`jQuery.ajax({
        url: '<%= repo_browser_assignment_submission_path(
          @assignment,
          id: @grouping.id,
          path: @path,
          revision_timestamp: @revision_timestamp,
          revision_number: @revision_number) %>',`